docs/billings-mt.md

```diff
@@ -0,0 +1,6 @@
+# Billings
+
+- The best thing about Billings is the view from the top of the Rimrocks, a sandstone bluff that runs along the north side of the Yellowstone valley.
+
+- You will drive up Zimmerman trail to the top of the Rims, and find a spot in one of the parks to pull over.
+- Sunsets are beautiful.
```

Comment:
In highschool we used to have keggers on the Rims, but now our favorite bonfire/keg location is covered with mini-mansions